recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "zipp" %}
-{% set version = "2.2.0" %}
+{% set version = "3.0.0" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5c56e330306215cd3553342cfafc73dda2c60792384117893f3a83f8a1209f50
+  sha256: 7c0f8e91abc0dc07a5068f315c52cb30c66bfbc581e5b50704c8a2f6ebae794a
   patches:
     # See https://github.com/jaraco/zipp/pull/30
```

Comment:
Could you please fix a dependency on ``host``?
``setuptools_scm`` should be ``setuptools_scm >=3.4.1``